Headers/DebugServer2/Target/Windows/Process.h

```diff
@@ -75,7 +75,7 @@ class Process : public ds2::Target::ProcessBase,
 
 public:
   ErrorCode enumerateSharedLibraries(
-      std::function<void(SharedLibraryInfo const &)> const &cb);
+      std::function<void(SharedLibraryInfo const &)> const &cb) override;
```

Comment:
This is good, without the `virtual`. Funny that this was never a virtual function in the first place.